cluster-default-publishing-strategy.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Cluster Publishing Strategy
+authors:
+  - "@abhinavdahiya"
+reviewers:
+  - "@openshift/api-reviewers"
+  - "@openshift/openshift-team-installer"
+  - "@ironcladlou"
+approvers:
+  - "@openshift/api-approvers"
+creation-date: 2019-09-18
+last-updated: 2019-09-18
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Cluster Publishing Strategy
+
+Administrators would like to be able to install OpenShift cluster as `internal/private`, which is only accessible from within the network, VPN tunnels and, not visible to the Internet. Therefore all the cluster operators that create endpoints for the cluster would need a place to discover this intent of the administrator and ensure that their resources are only accessible from within the network, VPN tunnels and not visible to the Internet by default.
```

Comment:
line breaks at about 120 chars so we can comment.

Installing is different than maintaining.  In the summary, you should describe the difference.